packages/renderer/src/lib/container/ContainerList.spec.ts

```diff
@@ -802,6 +802,9 @@ test('Sort containers based on selected parameter', async () => {
 
 test('Expect user confirmation to pop up when preferences require', async () => {
   vi.mocked(window.listContainers).mockResolvedValue([]);
+  vi.mocked(window.getConfigurationValue).mockResolvedValue(true);
+  Object.defineProperty(window, 'showMessageBox', { value: vi.fn() });
```

Comment:
```suggestion
```

Thsi is not needed

Reply:
The follow up PR is in draft here: #14940